src/material-experimental/mdc-select/select.spec.ts

```diff
@@ -222,6 +222,22 @@ describe('MDC-based MatSelect', () => {
           expect(select.getAttribute('tabindex')).toEqual('0');
         }));
 
+        it('should set `aria-describedby` to the id of the mat-hint', fakeAsync(() => {
+          expect(select.getAttribute('aria-describedby')).toBeNull();
+
+          fixture.componentInstance.hint = 'test';
+          fixture.detectChanges();
+          const hint = fixture.debugElement.query(By.css('mat-hint')).nativeElement;
+          expect(select.getAttribute('aria-describedby')).toBe(hint.getAttribute('id'));
```

Comment:
I think that you can drop this assertion since it's covered by the one below.

Reply:
This asserts that it's exactly the same as the hint's ID, whereas the next assertion asserts only that it matches a certain format: `expect(select.getAttribute('aria-describedby')).toMatch(/^mat-mdc-hint-\d+$/);`